### PR TITLE
Vector point refactor

### PR DESCRIPTION
**Pull request by abrahammurciano**
youre falling behind on code reviews

### CODE REVIEW

File: src/main/Main.java [hunk 1]
```diff
@@ -1,92 +0,0 @@
-import primitives.Vector;
```
Comment (elevin72) [inline src/main/Main.java:1]: Deleted forever? goooood

File: src/main/primitives/Point.java [hunk 2]
```diff
@@ -40,45 +18,27 @@
...
 	 * Adds a {@link Vector} to this {@link Point} and returns the resulting {@link Point}.
 	 *
-	 * @param v The {@link Vector} to add to this {@link Point}.
+	 * @param vector The {@link Vector} to add to this {@link Point}.
 	 * @return The {@link Point} resulting from adding the {@link Vector} to this {@link Point}.
 	 */
-	public Point add(Vector v) {
-		return transform((base, aux) -> base + aux, v.head);
+	public Point add(Vector vector) {
+		return new Point(transform(Double::sum, vector));
```
Comment (elevin72) [inline src/main/primitives/Point.java:41]: weird syntax bro

**Review by elevin72 — APPROVED** [review]
:+1:

File: src/main/primitives/Point.java (hunk 2 above)
Comment (abrahammurciano) [reply src/main/primitives/Point.java:41]: It's a function pointer to an existing function rather than reimplement it with a lambda

**Pull request merged**